.swiftlint.yml

```diff
@@ -1,7 +1,9 @@
 included: # paths to include during linting. `--path` is ignored if present.
```

Comment:
What are swiftlint rules?

Reply:
Good question 😂

Running it hasn't changed anything other than spacings, so for now I am tempted to class it as `working`, but you can start a separate discussion for changing the rules if you want 👍.

From [SwiftLint rules list](https://realm.github.io/SwiftLint/rule-directory.html) this configuration uses all the **Default Rules** and none of the **Opt-In Rules**. It then customises the default `identifier_name` rule to allow `id` as a valid property name (would normally be excluded on the grounds of being less than 3 characters long - default behaviour of that rule).